pkgs/desktops/gnome/games/gnome-2048/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchurl
+, wrapGAppsHook
+, meson
+, vala
+, pkg-config
+, ninja
+, itstool
+, clutter-gtk
+, libgee
+, gnome
+, gtk3
+}:
+
+let
+  pname = "gnome-2048";
```

Comment:
Also, due to Nix limitation, number after a dash is considered part of a version. We need to use `gnome-twenty-forty-eight` as the `pname`.